setup.py

```diff
@@ -14,6 +14,7 @@
 
 install_requires = [
     'draftjs_exporter==0.7.0',
+    'jinja2>=2.9.5,<3'  # TODO: remove when it's not necessary (https://github.com/springload/wagtaildraftail/issues/13)
```

Comment:
Wagtail implements everything for Django Templates by default and provide extra jinja2 tags in a [separate class](https://github.com/wagtail/wagtail/blob/1874b8b259e1295370b1c2122966d96668db7eec/wagtail/wagtailcore/jinja2tags.py#L10), which is not imported from anywhere by default. Then, if an implementation uses `Jinja2`, it can be [configured](https://github.com/wagtail/wagtail/blob/85de048a890aec4be3bcbf3af8a9ecfbf467d34a/wagtail/tests/settings.py#L69) to be included (and then it's up to the developer to make sure the right dependencies are installed). I think we should follow the same idea.